requirements.txt

```diff
@@ -25,17 +25,17 @@ pywin32==306; sys_platform == 'win32'
 pytesseract==0.3.7
 -e git+https://github.com/abrichr/pynput.git#egg=pynput
 pytest==7.1.3
+python-dotenv==1.0.0
+python-Levenshtein==0.21.1
 rapidocr-onnxruntime==1.2.3
 scikit-learn==1.2.2
 scipy==1.9.3
 setuptools-lint
+sounddevice~=0.4.6
+soundfile~=0.12.1
 sphinx
 sqlalchemy==1.4.43
-sumy==0.11.0
-tiktoken==0.4.0
 torch==2.0.0
 tqdm==4.64.0
-nicegui==1.2.16
 transformers==4.29.2
-python-dotenv==1.0.0
-python-Levenshtein==0.21.1
\ No newline at end of file
+git+https://github.com/openai/whisper.git
```

Comment:
Is this still the recommended way to install whisper?

I also found https://pypi.org/project/openai-whisper/

Reply:
`poetry add openai-whisper` isn't working for me as I get issues building wheel for llvmlite and numba, and also `Unable to find installation candidates for triton (2.0.0)`. The fixes online all suggest that this is because pip is older than version 19.0, but our pip is version 23.1.2. `poetry add git+https://github.com/openai/whisper.git` works so for now I'm just going with that.